supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/statuspolling/JesApiQueryManager.scala

```diff
@@ -72,44 +93,77 @@ class JesApiQueryManager(val qps: Int) extends Actor with ActorLogging {
   }
 
   private def handleTerminated(terminee: ActorRef) = {
-    // Currently we can assume this is a polling actor. Might change in a future update:
+    val cause = getFailureCause(terminee).getOrElse(new RuntimeException("No failure reason recorded"))
+    // We assume this is a polling actor. Might change in a future update:
```

Comment:
- The Polling actor dies
 - The supervisor puts it back into the queue with an incremented failure count

Same as any other failure, except it's resubmitted due to `Terminated(ref)`, rather than `JesApiFailure(...)`